src/Symfony/Component/Mime/MimeTypes.php

```diff
@@ -1249,7 +1278,7 @@ public function guessMimeType(string $path): ?string
         'application/xslt+xml' => ['xsl', 'xslt'],
         'application/xspf+xml' => ['xspf'],
         'application/xv+xml' => ['mxml', 'xhvml', 'xvml', 'xvm'],
-        'application/yaml' => ['yaml', 'yml'],
+        'application/yaml' => ['yml', 'yaml'],
```

Comment:
Let's not change this one as this is commonly used.

Reply:
Ok i understand where the problem is coming from.

I updated the issue comment and added  a comment in the update_mime_types.php file

(and fixed the MimeTypes.php file)